src/uuidV5.ts

```diff
@@ -0,0 +1,61 @@
+// uuidV5.ts
+
+/*
+ * Copyright (c) 2021 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
```

Comment:
We should make it clear this is derived from the uuid code, and include their license text in this file.

Reply:
Is there an example I could refer to?

Reply:
You need to have their license text put in here as-is, and maybe amend our text (above) to say it was based on url-to-uuid-repo

Reply:
updated. See if that's good enough.